lib/node_modules/@stdlib/blas/ext/base/sdssum/benchmark/benchmark.ndarray.js

```diff
@@ -21,14 +21,21 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/base/uniform' ).factory;
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float32Array = require( '@stdlib/array/float32' );
+
 var pkg = require( './../package.json' ).name;
 var sdssum = require( './../lib/ndarray.js' );
 
 
+
```

Comment:
```suggestion
```

Reply:
Please ensure you have your local dev environment setup, including running `make init`. Your PR contains various lint errors which project automated tooling can catch, but only if you have properly set things up locally.